### ISSUE TITLE
update angular-cli to beta 15

### ISSUE THREAD

**Issue by GabrielKatz** [angular-workshops/angular2-testing#1]
Hi,

I'm looking forward to the workshop next monday at AngularConnect!

Currently, there's an issue with angular-cli beta 14 and the new webpack version: https://github.com/angular/angular-cli/issues/2234

Could you please update the angular-cli in the package.json file?

Thanks

**Comment by joeeames** [comment]
are you actually experiencing an issue if you just clone the repo and npm install?

**Comment by GabrielKatz** [comment]
no, it only happens on npm start (guess I was a bit too curious...)

**Comment by joeeames** [comment]
wait, so you do get an error when you run npm start? it shouldn't error then. If it errors if you try to run ng new then that's fine, since we won't do that in class. but we'll definitely run npm start and it shouldn't error. can you screenshot the error you're getting? are you 100% sure it's the same issue as is related here? Updating the angular-cli is actually a bit of a pain. it's not as simple as just changing it in package.json unfortunately. it's several hours worth of work. so if things are stable then we don't want to mess with them. We are getting no problems running npm start (actually we're just running "ng serve" which is the same thing). So if you are we need to know about it.

**Comment by GabrielKatz** [comment]
You may close the issue. Peter updated the angular-cli yesterday with commit #d7fbe50.
